.vscode/controls.code-snippets

```diff
@@ -11,7 +11,7 @@
 			"    objective: <1-3 sentence description of the control objective>",
 			"    control_family: <Control Family>",
 			"    nist_csf: <NIST CSF control ID>",
-			"    threats:",
+			"    threat-mappings:",
```

Comment:
In the latest [schema](https://github.com/finos/common-cloud-controls/blob/84b4b8362efbac4af3a81111e8377b9623495e32/schemas/controls-schema.json#L104-L139) `threat-mappings` is a list of objects, not just ID strings. See example here: https://github.com/finos/common-cloud-controls/blob/84b4b8362efbac4af3a81111e8377b9623495e32/catalogs/networking/loadbalancer/controls.yaml#L62-L70